agenthub/codeact_agent/codeact_agent.py

```diff
@@ -133,23 +133,22 @@ def get_action_message(self, action: Action) -> Message | None:
 
     def get_observation_message(self, obs: Observation) -> Message | None:
         max_message_chars = self.llm.config.max_message_chars
-        obs_prefix = 'OBSERVATION:\n'
+        obs_prefix = r'OBSERVATION:\n'
```

Comment:
Any idea why we need to make these `\n` escaped?

![Uploading image.png…]()

Reply:
That's already out of my local source. Call it v0.1. :)